pkg/ucp/frontend/api/routes.go

```diff
@@ -37,8 +34,8 @@ import (
 )
 
 const (
-	planeCollectionPath       = "/planes"
-	planeCollectionByTypePath = "/planes/{planeType}"
+	planeCollectionPath      = "/planes"
+	planesTypeCollectionPath = "/planes/{planeType}"
```

Comment:
Any reason why this is plural and the other is singular?

Reply:
I think just a typo, I'll change it 👍